schemas/generated/data.json

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
If this is generated, do we want it committed? Should it instead be an artifact resulting from CI?

Reply:
if I don't commit it manually here, it won't be generated until the next time the TOMLs are updated